resources/lib/ui/control.py

```diff
@@ -143,6 +143,14 @@ def multiselect_dialog(title, _list):
         return xbmcgui.Dialog().multiselect(title, _list)
     return None
 
+def set_property(name, value):
```

Comment:
This feels very hacky solution for passing a parameter.are you sure this is mandatory and not just about passing a param to play to set autoplay false, but only once?